ci-operator/step-registry/ipi/conf/capability/baselinecaps/vset/ipi-conf-capability-baselinecaps-vset-commands.sh

```diff
@@ -84,12 +103,29 @@ capabilities:
 EOF
 
 #To enable required capablities whatever baselineCapabilitySet setting
+additional_caps=""
 if [[ "${ADDITIONAL_ENABLED_CAPABILITIES}" != "" ]]; then
     echo "Enable required capabilities: ${ADDITIONAL_ENABLED_CAPABILITIES}"
+    additional_caps="${ADDITIONAL_ENABLED_CAPABILITIES}"
+fi
+
+# Capablities dependency
+if [[ "${selected_cap_set}" != "vCurrent" ]]; then
+    # cap marketplace must be enabled along with OperatorLifecycleManager
+    selected_set=${selected_cap_set//.}
+    for key in "${!dependency_caps[@]}"; do
+        if [[ "${!selected_set}" =~ ${key} ]] && [[ ! "${!selected_set}" =~ ${dependency_caps[$key]} ]] && [[ "${v_current}" =~ ${dependency_caps[$key]} ]]; then
```

Comment:
```suggestion
        if [[ " ${!selected_set} " =~ " ${key} " ]] && [[ ! " ${!selected_set} " =~ " ${dependency_caps[$key]} " ]] && [[ " ${v_current} " =~ " ${dependency_caps[$key]} " ]]; then
```
Maybe add the leading whitespace and tail whitespace will make the match more safe and accurate.